packages/core/src/rum/DdRum.ts

```diff
@@ -130,6 +130,58 @@ class DdRumWrapper implements DdRumType {
         return this.callNativeStopAction(...nativeCallArgs);
     };
 
+    startFeatureOperation(
+        name: string,
+        operationKey: string | null,
+        attributes: object
+    ): Promise<void> {
+        InternalLog.log(
+            `Starting feature operation “${name}” (${operationKey})`,
+            SdkVerbosity.DEBUG
+        );
+        return bufferVoidNativeCall(() =>
+            this.nativeRum.startFeatureOperation(name, operationKey, attributes)
+        );
+    }
+
+    succeedFeatureOperation(
+        name: string,
+        operationKey: string | null,
+        attributes: object
+    ): Promise<void> {
+        InternalLog.log(
+            `Succeding feature operation “${name}” (${operationKey})`,
+            SdkVerbosity.DEBUG
+        );
+        return bufferVoidNativeCall(() =>
+            this.nativeRum.succeedFeatureOperation(
+                name,
+                operationKey,
+                attributes
+            )
+        );
+    }
+
+    failFeatureOperation(
+        name: string,
+        operationKey: string | null,
+        reason: string, // TODO: update type
```

Comment:
Is this a leftover or is it a future improvement?, if it's the second case I think we should attach a Ticket ID to tackle it.